system/Throttle/Throttler.php

```diff
@@ -171,7 +169,7 @@ public function check(string $key, int $capacity, int $seconds, int $cost = 1):
 
 		// If $tokens > 0, then we are safe to perform the action, but
 		// we need to decrement the number of available tokens.
-		if ($tokens > 0)
+		if ($tokens >= 0)
```

Comment:
comment need to be updated

Reply:
Yes, I am updating both comments and checking errors in unit tests.